src/Sitewards/DBCompare/Question/DBQuestion.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Sitewards\DBCompare\Question;
+
+use Symfony\Component\Console\Helper\QuestionHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+
+class DBQuestion
+{
+    /** @var QuestionHelper */
+    private $oQuestionHelper;
+
+    /**
+     * @param QuestionHelper $oQuestionHelper
+     */
+    public function __construct(
+        QuestionHelper $oQuestionHelper
+    ) {
+        $this->oQuestionHelper = $oQuestionHelper;
+    }
+
+    /**
+     * @param InputInterface $oInput
+     * @param OutputInterface $oOutput
+     * @param string $sQuestion
+     * @return string
+     * @throws \Symfony\Component\Console\Exception\RuntimeException
+     */
+    public function getBasicInformation(
+        InputInterface $oInput,
+        OutputInterface $oOutput,
+        $sQuestion = ''
+    ) {
+        $oDBInfoQuestion = new Question($sQuestion);
```

Comment:
Shouldn't we create this object when we call this method? this way we can mock the object to achieve certain behavior during tests...